Makefile_packaging.mk

```diff
@@ -210,22 +211,87 @@ debs: $(DEBS)
 ls: $(TARGETS)
 	ls -ld $^
 
-ifneq ($(ID_LIKE),suse)
+ifeq ($(ID_LIKE),rhel fedora)
 chrootbuild: $(SRPM) Makefile
-
+	echo -e "config_opts['yum.conf'] += \"\"\"\n" >> /etc/mock/default.cfg;  \
+	for repo in $(ADD_REPOS); do                                                 \
+	    if [[ $$repo = *@* ]]; then                                          \
+	        branch="$${repo#*@}";                                            \
+	        repo="$${repo%@*}";                                              \
+	    else                                                                 \
+	        branch="master";                                                 \
+	    fi;                                                                  \
+	    echo -e "[$$repo:$$branch:lastSuccessful]\n\
+name=$$repo:$$branch:lastSuccessful\n\
+baseurl=$${JENKINS_URL}job/daos-stack/job/$$repo/job/$$branch/lastSuccessfulBuild/artifact/artifacts/centos7/\n\
+enabled=1\n\
+gpgcheck = False\n" >> /etc/mock/default.cfg;                                    \
+	done;                                                                    \
+	echo "\"\"\"" >> /etc/mock/default.cfg
 	mock $(MOCK_OPTIONS) $(RPM_BUILD_OPTIONS) $<
 else
-ADD_REPOS := $(shell if [ -e /tmp/build.repos ]; then cat /tmp/build.repos; fi)
-chrootbuild: Makefile $(SOURCES)
-	zypper lr --details
-	sudo zypper --non-interactive --no-gpg-checks refresh
-	sudo build $(BUILD_OPTIONS) $(ADD_REPOS) --repo zypp:// \
-	  --dist $(DISTRO_ID) $(RPM_BUILD_OPTIONS)
+sle12_REPOS += --repo https://download.opensuse.org/repositories/science:/HPC/openSUSE_Leap_42.3/     \
+	       --repo http://cobbler/cobbler/repo_mirror/sdkupdate-sles12.3-x86_64/                 \
+	       --repo http://cobbler/cobbler/repo_mirror/sdk-sles12.3-x86_64                        \
+	       --repo http://download.opensuse.org/repositories/openSUSE:/Backports:/SLE-12/standard/ \
+	       --repo http://cobbler/cobbler/repo_mirror/updates-sles12.3-x86_64                    \
+	       --repo http://mgmt-1.wolf.hpdd.intel.com/cobbler/pub/SLES-12.3-x86_64/
```

Comment:
Agreed.  But this really only applies to SLES since Leap repos are publicly available.  So we'll probably end up with an environment variable (or some such) that points to one's local server for SLES repos and just skip building them if not set.